src/tests/integration/test-data/src/Unit.t.sol

```diff
@@ -181,9 +181,17 @@ contract UnitTest is Test {
 
     function test_assertApproxEqAbs_uint(uint256 a, uint256 b, uint256 maxDelta) public pure {
         vm.assume(maxDelta >= max(a, b) - min(a, b));
-        string memory err = "throw test";
         assertApproxEqAbs(a, b, maxDelta);
-        assertApproxEqAbs(a, b, maxDelta, err); 
+    }
+
+    function test_assertApproxEqAbs_uint_err() public {
+        string memory err = "throw test";
+        string memory err_lt = "throw test: 11 !~= 121 (max delta: 100, real delta: 110)";
+        string memory err_gt = "throw test: 121 !~= 11 (max delta: 100, real delta: 110)";
+        vm.expectRevert(bytes(err_lt));
```

Comment:
That's not directly related to this PR, but I'm not sure how well Kontrol handles tests with several `expectRevert`s, might be worth checking.

Reply:
Okay. Will check on this.

Reply:
I have limited all `test_*_err` tests to have 1 `expectRevert` in commit https://github.com/runtimeverification/kontrol/pull/593/commits/b5064de4b17f7383531ffe138ffe964e6e1842ed.

Reply:
Thanks! Does it still fail with the same issue? Do you think we support several subsequent `expectRevert`s in general?